### ISSUE TITLE
[FEATURE]

### ISSUE THREAD

**Issue by subdavis** [VIAME/VIAME-Web#351]
**Is your feature request related to a problem? If so, Please describe.**
It's overkill to go through a whole pipeline just to generate some JSON

**Describe the solution you'd like**
Perform full-frame detection generation client-side, remove utility pipelines from UI.

Another note:  IMO, all detections in the sequence should be part of the same track, not separate tracks.  This will make it a lot less difficult to rapidly annotate full frames: you don't have to click/select every time you move frames.

That way the track field can be used to set attributes on the whole dataset, and the detection attributes can be used like normal.  Otherwise, it's confusing whether you should use a track or a detection attribute for the full frame attribute.

**Comment by waxlamp** [comment]
@subdavis is this still valid?

**Comment by subdavis** [comment]
Yeah, it would be nice to have, but it's Low or no priority.  We should keep the issue though.
